src/index.ts

```diff
@@ -77,10 +77,14 @@ export class MaticPOSClient extends SDKClient {
     if (!txHash) {
       throw new Error(`txHash not provided`)
     }
-    if (options && !options.from) {
-      throw new Error(`options.from is missing`)
+    if (options && !options.from && !options.fastProof) {
+      throw new Error(`from or fastProof is missing`)
+    }
+    if (options.fastProof) {
+      return this.posRootChainManager.exitERC20Hermoine(txHash, options)
+    } else {
+      return this.posRootChainManager.exitERC20(txHash, options)
     }
-    return this.posRootChainManager.exitERC20(txHash, options)
   }
 
   exitERC20Token(txHash: string, options?: SendOptions) {
```

Comment:
Remove this function since `exitERC20` can be used for both normal and hermione. Same applies to other instances also